numpy/tests/test_matlib.py

```diff
@@ -1,11 +1,3 @@
-# As we are testing matrices, we ignore its PendingDeprecationWarnings
-try:
-    import pytest
-    pytestmark = pytest.mark.filterwarnings(
```

Comment:
I tried adding my ignore here and it didn't suprress so I'm not sure this code works even if it wasn't covered by pytest.ini and _pytesttester.py

Reply:
This code did nothing, so removing it is fine.